mlflow/entities/trace_status.py

```diff
@@ -1,12 +1,13 @@
 from mlflow.protos.service_pb2 import TraceStatus as ProtoTraceStatus
+from mlflow.tracing.types.constant import TraceStatusCode
 
 
 class TraceStatus:
     """Enum for status of an :py:class:`mlflow.entities.TraceInfo`."""
 
-    UNSPECIFIED = ProtoTraceStatus.Value("TRACE_STATUS_UNSPECIFIED")
-    OK = ProtoTraceStatus.Value("OK")
-    ERROR = ProtoTraceStatus.Value("ERROR")
+    UNSPECIFIED = ProtoTraceStatus.Value(TraceStatusCode.UNSPECIFIED)
```

Comment:
I'm a bit confused about `TraceStatus` and `TraceStatusCode`, why do we need both?

Reply:
`TraceStatus` enum values are integer (coming from proto value), while we wanna let users to specify status with string. Alternatively, we can add `TraceStatusCode` as a unified entity, which has `to_proto()` method, WDYT?